app/views/past_foreign_secretaries/_show_person.html.erb

```diff
@@ -0,0 +1,102 @@
+<% content_for :breadcrumbs do %>
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <div class="gem-c-layout-for-public__blue-bar"></div>
+        <%= render "govuk_publishing_components/components/breadcrumbs", {
+          breadcrumbs: [
+            {
+              title: "Home",
+              url: "/"
+            },
+            {
+              title: "History of the UK Government",
+              url: "/government/history"
+            }
+          ]
+        } %>
+      </div>
+    </div>
+  </div>
+<% end %>
```

Comment:
Do we need this, since the usage of `prefix` in the content item means we should retrieve the content item for the `/government/past-foreign-secretaries` page (which has the breadcrumbs in the links)?

Reply:
Oh yes good point, this can go now